CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+* @generalassembly/infrastructure
+
+# Base config files
+dev_tools/rubocop/base.yml @generalassembly/infrastructure @generalassembly/edtech @generalassembly/martech @generalassembly/engineering-support
+dev_tools/rubocop/rails-base.yml @generalassembly/infrastructure @generalassembly/edtech @generalassembly/martech @generalassembly/engineering-support
+dev_tools/rubocop/rpsec-base.yml @generalassembly/infrastructure @generalassembly/edtech @generalassembly/martech @generalassembly/engineering-support
+
+# Infrastructure config files
+dev_tools/rubocop/infrastructure.yml @generalassembly/infrastructure
+dev_tools/rubocop/infrastructure-rails.yml @generalassembly/infrastructure
+
+# Edtech config files
+dev_tools/rubocop/edtech.yml @generalassembly/edtech
+dev_tools/rubocop/edtech-rails.yml @generalassembly/edtech
+
+# Martech config files
+dev_tools/rubocop/martech.yml @generalassembly/martech
+dev_tools/rubocop/martech-rails.yml @generalassembly/martech
```

Comment:
I'd like to confirm the intended behavior for reviewers:

* in general Infra is added as reviewers for any changes (`*` rule)
* in a PR where only the team-specific files are modified, _only_ those teams are added as reviewers and not Infra

Is this correct?

Reply:
Yeah, that was my intention